src/python/pants/backend/python/dependency_inference/module_mapper.py

```diff
@@ -202,7 +202,7 @@ def providers_for_module(
 async def merge_first_party_module_mappings(
     union_membership: UnionMembership,
 ) -> FirstPartyPythonModuleMapping:
-    all_mappings = await MultiGet(
+    all_mappings = await concurrently(
         Get(
```

Comment:
Remaining Get in this file - I think it's a union rule (or other, non-mechanical change) - deferring this

Reply:
Yup, I believe this is a union to allow dependency inference implementations to hook in.